src/main/java/software/sigma/internship/dto/TeacherDto.java

```diff
@@ -3,11 +3,14 @@
 import lombok.Data;
 import software.sigma.internship.entity.Teacher;
 
+import java.util.List;
+
 
 @Data
 public class TeacherDto {
     private Long id;
     private String firstName;
     private String lastName;
     private Teacher.Position position;
-}
+    private List<TestDto> tests;
```

Comment:
I am not sure that you need a list of tests in Teacher. Let's discuss it.

Reply:
I have removed id and made a new method in controller to find tests that belong to one teacher, please check my last commit.